package.json

```diff
@@ -14,11 +14,13 @@
   "dependencies": {
     "chalk": "^5.3.0",
     "commander": "^12.1.0",
+    "express": "^4.21.2",
     "figlet": "^1.7.0",
     "fs-extra": "^11.2.0",
-    "nanospinner": "^1.1.0"
+    "nanospinner": "^1.1.0",
+    "supertest": "^7.0.0"
   },
   "devDependencies": {
     "jest": "^29.7.0"
   }
-}
\ No newline at end of file
+}
```

Comment:
`express` and `supertest` are template specific. Add it to template's `package.json`. Create one for `basic` as it doesn't exist. Refer latest changes from #57.